pkg/sql/schemachanger/scdecomp/decomp.go

```diff
@@ -571,6 +592,13 @@ func (w *walkCtx) walkIndex(tbl catalog.TableDescriptor, idx catalog.Index) {
 		TableID: tbl.GetID(),
 		IndexID: idx.GetID(),
 	})
+	if !idx.GetGeoConfig().IsEmpty() {
+		w.ev(scpb.Status_PUBLIC, &scpb.IndexGeoConfig{
+			TableID: tbl.GetID(),
+			IndexID: idx.GetID(),
+			Config:  idx.GetGeoConfig(),
+		})
+	}
```

Comment:
Why is `IndexGeoConfig` its own element instead of the geo config being part of `Index`? It's not something that can be subject to schema changes independently of an index, AFAIK.